src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -193,16 +182,18 @@ protected void runTool( final JavaSparkContext ctx ) {
         final HopscotchUniqueMultiMap<String, Integer, QNameAndInterval> qNamesMultiMap = intervalsAndQNameMap._2;
 
         // supplement the template names with other reads that share kmers
-        if ( !intervalOnlyAssembly )
-            addAssemblyQNames(params, ctx, kmersToIgnoreFile, qNamesMultiMap, allPrimaryLines, locations, pipelineOptions);
+        final List<Tuple2<Integer, String>> intervalDispositions;
```

Comment:
Why not make intervalDispositions a `Map<Integer, String>`? Memory shouldn't be too big a factor with the number of intervals we consider, and to my eye it would be easier to read.